packages/remix-codemod/bin/cli/check-git-status.ts

```diff
@@ -0,0 +1,39 @@
+import isGitClean from "is-git-clean";
```

Comment:
I _think_ this file is copied from [`react-codemod`](https://github.com/reactjs/react-codemod/blob/master/bin/cli.js#L22).

Could you include a file-level comment with a link to the original and stating if this purely copied or if there were modifications from the original?

^I'd request the same for any other files in `cli/` that were taken from `react-codemod`, just to make reviewing this a bit easier 😃